### ISSUE TITLE
[Azure Toolkit] [A11yMAS][Accessibility][Keyboard][MAS2.4.3][Eclipse][Wipro]:Keyboard focus order is illogical when create new web app radio button is selected in run on web app for containers window 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1530]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app for containers.
4. "Run on web app for containers" dialog, Select "web app for container->create New Radio button ->Tab to "resource Group"->Press down arrow->App Service Plan then press tab and observe the keyboard focus

**Expected Result:**
Keyboard focus should be on " Use existing radio button" which is present under "App Service Plan"

**Actual Result:**
Keyboard focus is going to use existing radio button which is present under "Resource group"
**observation:** In the same window tab up to "Subscription" combo box  and press tab then observe the keyboard focus, keyboard focus is going to App service plan instead of resource group.

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![keyboard focus order is illogical in web app for container](https://user-images.githubusercontent.com/22956130/35681359-212c548a-0783-11e8-8390-befb93d98846.jpg)

**Comment by Eskibear** [comment]
under investigation

**Comment by Eskibear** [comment]
`ExpandBar` , it's determined by Eclipse. It's out of toolkit's control.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533977

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. It's a legacy feature hardly ever used.
* **Quality of experience**: not impacted. A simple workaround is to use the Azure portal to create related resources and deploy.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

#### Workaround
![image](https://user-images.githubusercontent.com/2351748/47149111-3c041d80-d305-11e8-94f3-1e975d849ba7.png)

**Comment by umasarath52** [comment]
Discussed with Ryan and approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749963
